pkg/controller/memberstatus/memberstatus_controller.go

```diff
@@ -166,8 +167,11 @@ func (r *ReconcileMemberStatus) hostConnectionHandleStatus(reqLogger logr.Logger
 	}
 
 	// look up host connection status
-	clusterStatus, err := status.GetKubefedConditions(attributes)
-	memberStatus.Status.HostConnection = clusterStatus
+	connectionConditions := status.GetKubefedConditions(attributes)
```

Comment:
I'm late on this, but the PR description says

> This PR updates the member status controller to use the new memberStatus.Status.Host field which now uses Toolchain Conditions instead of Kubefed conditions.

but we're still using `status.GetKubefedConditions(attributes)`? I'm probably missing something 🤔